core/web/jobs_controller_test.go

```diff
@@ -182,12 +181,16 @@ func TestJobController_Create_HappyPath(t *testing.T) {
 		},
 		{
 			name: "keeper",
-			toml: testspecs.GenerateKeeperSpec(testspecs.KeeperSpecParams{
-				Name:              "example keeper spec",
-				ContractAddress:   "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba",
-				FromAddress:       "0xa8037A20989AFcBC51798de9762b351D63ff462e",
-				ObservationSource: keeper.ExpectedObservationSource,
-			}).Toml(),
+			toml: `
+                                  type                        = "keeper"
+                                  schemaVersion               = 3
```

Comment:
do we really use schemaVersion? We seem to accept '3' here when we shouldn't.

But it's probably a different task to look at how schemaVersion is used holistically.

Reply:
For this test case in particular schemaVersion is not required indeed, but to be honest I would have to do a bit more research to understand its role in a wider scope.

Reply:
yup agree, it's a different ticket :)